AK/Tuple.h

```diff
@@ -221,6 +221,26 @@ Tuple(Args... args) -> Tuple<Args...>;
 
 }
 
+namespace std {
+
+template<class T>
+struct tuple_size;
+
+template<size_t index, class T>
+struct tuple_element;
+
+}
```

Comment:
It's not legal to forward declare these...

If the stdlib is libc++, these will actually be declared in `_LIBCPP_NAMESPACE_STD` which defaults to `namespace std { inline namespace __v1 {`, or other names on weird platforms like Android where there's a 'system' libc++ and a 'user' libc++ in different ABI namespaces.

What happens if you compile a source file that includes both `AK/Tuple.h` and `<tuple>` ? with both libc++ and libstdc++. 

Have a look here: https://godbolt.org/z/9c78dE6EY